### ISSUE TITLE
Shortcut to reselect selection

### ISSUE THREAD

**Issue by joshpetit** [flameshot-org/flameshot#3319]
### Feature Description

When you have a current selection, you can press a button to remove that selection and be prompted to select a part of the screen again. 

I looked through the keyboard shortcuts [here](https://flameshot.org/docs/guide/key-bindings/) and couldn't figure out if this feature exists already

**Comment by mmahmoudian** [comment]
Thanks for the feature request, but I'm a bit confused and need more clarification. Are you looking for
1.  a shortcut to cancel a selection
2. a shortcut to undo cancellation 
3. A shortcut to select a region that was used in the previous screenshot

**Comment by joshpetit** [comment]
The first one pretty much. Cancels the current region and allows you to select a new region.

I'd like this in particular since I have the "reselect last region" option on, so if I could change the region that would be nice.

**Comment by arunnalpet** [comment]
+1 for "shortcut to select a region that was used in the previous snapshot".. 

For example, if you are capturing screenshots from video.. the region will be mostly fixed.. you would want to capture the same region, multiple times.. So need an option to quickly select a region that was used in previous region.. Instead of drawing the region every time..

**Comment by joshpetit** [comment]
> +1 for "shortcut to select a region that was used in the previous snapshot".. 
> 
> For example, if you are capturing screenshots from video.. the region will be mostly fixed.. you would want to capture the same region, multiple times.. So need an option to quickly select a region that was used in previous region.. Instead of drawing the region every time.. 

This already exists, it's in the settings

**Comment by mmahmoudian** [comment]
@arunnalpet 

As josh kindly pointed out, this is already possible:

![image](https://github.com/flameshot-org/flameshot/assets/390889/ab570bba-5515-43aa-964e-3d89a73662b2)


@joshpetit 
> I'd like this in particular since I have the "reselect last region" option on, so if I could change the region that would be nice.

Just to be 100% sure of what you are looking for, you want to a way to deselect the region, and then be able to select a new region. Right?

I think this would also solve one more problem: is someone do <kbd>Ctrl</kbd><kbd>a</kbd> and take a screenshot, and have the "Use last region" option on, there is no way for them to cancel the selection.

Would it be enough if pressing <kbd>Esc</kbd> cancels he selected region?

**Comment by joshpetit** [comment]
> Just to be 100% sure of what you are looking for, you want to a way to deselect the region, and then be able to select a new region. Right?
> 
> I think this would also solve one more problem: is someone do Ctrla and take a screenshot, and have the "Use last region" option on, there is no way for them to cancel the selection.
> 
> Would it be enough if pressing Esc cancels he selected region?

Yup, deselecting the region is exactly what I'm speaking about. I would be wary to change escape to cancel the last selected region since that would change how it's worked since forever. I'd say possibly make an option that says "Escape cancels selection" rather. The Backspace key seems to be unused though, so that could be the button.

All up to you though, I'd be fine with whatever haha. Thank you.
